Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -222,7 +225,13 @@ public class PBXProjGenerator {
             pbxProject.projects = subprojects
         }
 
-        try project.targets.forEach(generateTarget)
+        // TODO: Generate targets concurrently
+        // At the moment, there's some non-determinism involved, leading to different framework orders in the
```

Comment:
@yonaskolb do you know where these are added? Sorting them at the end would probably fix this.